src/atoms/Trackable/__tests__/trackable.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+
+import Trackable from 'atoms/Trackable';
+import { dataAttributes, dataAttrName } from 'atoms/Trackable/utils';
+
+describe('<Trackable />', () => {
+
+  describe('dataAttributes()', () => {
+
+    it('takes a string and returns a lower+kebab case prefixed with data-', () => {
+      expect(dataAttrName('SoMe Name')).to.be.equal('data-some-name');
```

Comment:
Is this supposed to be "SoMe"? What happens when you don't use the correct casing?